app/controllers/v0/benefits_claims_controller.rb

```diff
@@ -26,8 +27,27 @@ def submit5103
 
     private
 
+    def claims_scope
+      EVSSClaim.for_user(@current_user)
+    end
+
     def service
       @service ||= BenefitsClaims::Service.new(@current_user.icn)
     end
+
+    def tap_claims(claims)
+      claims.each do |claim|
+        record = claims_scope.where(evss_id: claim['id']).first
+
+        if record.blank?
+          EVSSClaim.create(
+            user_uuid: @current_user.uuid,
+            user_account: @current_user.user_account,
+            evss_id: claim['id'],
+            data: {}
```

Comment:
What's the purpose of `data` here? Why can we leave this blank?

Reply:
IIRC It's the field that ends up holding all of the JSON that gets returned from EVSS when we request the claim details later

Reply:
https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/workers/evss/retrieve_claims_from_remote_job.rb#L35-L38

Reply:
I took the code from here and changed the method from `new` to `create`